deployment/cdk/opensearch-service-migration/lib/stack-composer.ts

```diff
@@ -435,6 +435,7 @@ export class StackComposer {
             reindexFromSnapshotStack = new ReindexFromSnapshotStack(scope, "reindexFromSnapshotStack", {
                 vpc: networkStack.vpc,
                 extraArgs: reindexFromSnapshotExtraArgs,
+                clusterAuthDetails: servicesYaml.target_cluster,
```

Comment:
When is target_cluster populated, do we need to introduce a stack dependency to make sure it's populated by this deployment?